components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/OAuth2ServiceTest.java

```diff
@@ -470,4 +631,88 @@ private void setUpRevokeToken() throws Exception {
         doNothing().when(OAuthUtil.class, "clearOAuthCache", anyString(), anyString(), anyString());
     }
 
+    @Test
+    public void testGetOauthApplicationState() throws Exception {
+
+        String id = "clientId1";
+        OAuthAppDO oAuthAppDO = new OAuthAppDO();
+        oAuthAppDO.setState("ACTIVE");
+        whenNew(OAuthAppDAO.class).withNoArguments().thenReturn(oAuthAppDAO);
+        when(oAuthAppDAO.getAppInformation(id)).thenReturn(oAuthAppDO);
+        mockStatic(IdentityTenantUtil.class);
+        when(IdentityTenantUtil.getTenantId(anyString())).thenReturn(1);
+
+        assertEquals(oAuth2Service.getOauthApplicationState(id), "ACTIVE");
+    }
+
+    @Test
+    public void testGetOauthApplicationStateWithIdentityOAuth2Exception() throws Exception {
+
+        mockStatic(IdentityTenantUtil.class);
+        when(IdentityTenantUtil.getTenantId(anyString())).thenReturn(1);
+        whenNew(OAuthAppDAO.class).withNoArguments().thenThrow(IdentityOAuth2Exception.class);
+        assertNull(oAuth2Service.getOauthApplicationState(clientId));
+    }
+
+    @Test
+    public void testGetOauthApplicationStateWithInvalidOAuthClientExceptionException() throws Exception {
```

Comment:
The word 'Exception' has been repeated in the method name. Please fix.

Reply:
Addressed with commit [9e575a3](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1691/commits/9e575a323c0604e9985d430a5b3fee4942d86cb1)